PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/webapp/webappconfig/slimui/creation/WebAppCreationDialog.java

```diff
@@ -0,0 +1,483 @@
+    private void selectAppServicePlan() {
+        AppServicePlan appServicePlan = (AppServicePlan) cbExistAppServicePlan.getSelectedItem();
+        if (appServicePlan == null) {
+            lblLocation.setText(NOT_APPLICABLE);
+            lblPricing.setText(NOT_APPLICABLE);
+        } else {
+            lblLocation.setText(appServicePlan.regionName());
+            lblPricing.setText(appServicePlan.pricingTier().toString());
+        }
+    }
+
+    private void onOK() {
+        createWebApp();
+    }
+
+    public WebApp getCreatedWebApp() {
+        return this.result;
+    }
+
+    private void updateConfiguration() {
+        webAppConfiguration.setWebAppName(txtWebAppName.getText());
+        webAppConfiguration.setSubscriptionId(cbSubscription.getSelectedItem() == null ? "" :
+            ((Subscription) cbSubscription.getSelectedItem()).subscriptionId());
+        // resource group
+        if (rdoCreateResGrp.isSelected()) {
+            webAppConfiguration.setCreatingResGrp(true);
+            webAppConfiguration.setResourceGroup(txtNewResGrp.getText());
+        } else {
+            webAppConfiguration.setCreatingResGrp(false);
+            webAppConfiguration.setResourceGroup(cbExistResGrp.getSelectedItem() == null ? "" :
+                ((ResourceGroup) cbExistResGrp.getSelectedItem()).name());
+        }
+        // app service plan
+        if (rdoCreateAppServicePlan.isSelected()) {
+            webAppConfiguration.setCreatingAppServicePlan(true);
+            webAppConfiguration.setAppServicePlanName(txtCreateAppServicePlan.getText());
+            webAppConfiguration.setRegion(cbLocation.getSelectedItem() == null ? DEFAULT_REGION.name() :
+                ((Location) cbLocation.getSelectedItem()).region().name());
+            webAppConfiguration.setPricing(cbPricing.getSelectedItem() == null ? DEFAULT_PRICINGTIER.toString() :
+                cbPricing.getSelectedItem().toString());
+        } else {
+            webAppConfiguration.setCreatingAppServicePlan(false);
+            AppServicePlan appServicePlan = (AppServicePlan) cbExistAppServicePlan.getSelectedItem();
+            if (appServicePlan != null) {
+                webAppConfiguration.setAppServicePlanId(appServicePlan.id());
+            }
+        }
+        // runtime
+        if (rdoLinuxOS.isSelected()) {
+            webAppConfiguration.setOS(OperatingSystem.LINUX);
+            RuntimeStack linuxRuntime = (RuntimeStack) cbRuntimeStack.getSelectedItem();
+            if (linuxRuntime != null) {
+                webAppConfiguration.setStack(linuxRuntime.stack());
+                webAppConfiguration.setVersion(linuxRuntime.version());
+            }
+        }
+        if (rdoWindowsOS.isSelected()) {
+            webAppConfiguration.setOS(OperatingSystem.WINDOWS);
+            JdkModel jdkModel = (JdkModel) cbJdkVersion.getSelectedItem();
+            if (jdkModel != null) {
+                webAppConfiguration.setJdkVersion(jdkModel.getJavaVersion());
```

Comment:
If jdkModel may be null, here cannot cast directly

Reply:
added null check, thanks~